project_cost_report_supply/report.py

```diff
@@ -0,0 +1,102 @@
+# © 2019 Numigi (tm) and all its contributors (https://bit.ly/numigiens)
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models, _
+from odoo.osv.expression import AND
+from odoo.tools.float_utils import float_round
+from odoo.addons.project_cost_report.report import CostReportCategory
+
+
+class ProjectCostReportWithShopSupply(models.TransientModel):
+
+    _inherit = 'project.cost.report'
+
+    def _get_empty_shop_supply_category_label(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method could be a function](https://app.codacy.com/app/numigi/odoo-project-addons/pullRequest?prid=4044060)